### DISCUSSION TITLE
Show Connection information in Result Tab

### DISCUSSION THREAD

**Discussion by g-industrialist** [dbeaver/dbeaver#22618] [DBeaver Development]
Result Tab is a good idea / function that other paid IDE tool does not provide. However it lacks connection (say which DB environment that Result Set is from).

**Comment by LonwoLonwo** [comment]
Hello @iseehk 

Did you check the tooltip? I can see connection info there:

![2024-01-29 17_31_03-DBeaver Ultimate 23 3 4 - _dev 8_ Script-170](https://github.com/dbeaver/dbeaver/assets/45152336/3b6c809d-7dd6-4508-a117-3fbb9c22350c)

**Comment by g-industrialist** [comment]
@LonwoLonwo  Thank you , 
Sorry, I missed it.
